uaclient/livepatch.py

```diff
@@ -210,24 +229,21 @@ def is_kernel_supported(
             )
             return None
 
-        return bool(result.get("Supported", False))
+        api_supported_val = result.get("Supported")
+        if api_supported_val is None or isinstance(api_supported_val, bool):
+            # old version, True means supported, None means unsupported
+            if api_supported_val:
+                return LivepatchSupport.SUPPORTED
+            return LivepatchSupport.UNSUPPORTED
+        # new version, value is a string
+        return _convert_str_to_livepatch_support_status(api_supported_val)
```

Comment:
@kian99 For reference, as discussed, this is how we're going to handle both versions of the API. Once the new version hits production, we can remove the compatibility for the old version

Reply:
Gotcha, appreciate Pro client handling this oversight from our side.